### ISSUE TITLE
az acs kubernetes install-cli overwrites ~/.kube/config

### ISSUE THREAD

**Issue by bnookala** [Azure/azure-cli#5381]
I just noticed that attempting to install a new version of `kubectl` from `az acs kubernetes install-cli` overwrites the kubeconfig in ~/.kube/config. Environment and reproduction below:

---

### Environment summary

Install Method: Homebrew
OS: Mac OS 10.13.2 
Shell: zsh
Cli version:  2.0.25
Components: 
- acr (2.0.19)
- acs (2.0.24)
- advisor (0.1.1)
- appservice (0.1.24)
- backup (1.0.6)
- batch (3.1.8)
- batchai (0.1.4)
- billing (0.1.7)
- cdn (0.0.11)
- cloud (2.0.11)
- cognitiveservices (0.1.10)
- command-modules-nspkg (2.0.1)
- configure (2.0.13)
- consumption (0.2.1)
- container (0.1.16)
- core (2.0.25)
- cosmosdb (0.1.16)
- dla (0.0.17)
- dls (0.0.19)
- eventgrid (0.1.8)
- extension (0.0.7)
- feedback (2.0.8)
- find (0.2.8)
- interactive (0.3.13)
- iot (0.1.16)
- keyvault (2.0.16)
- lab (0.0.15)
- monitor (0.1.0)
- network (2.0.21)
- nspkg (3.0.1)
- profile (2.0.17)
- rdbms (0.0.11)
- redis (0.2.11)
- reservations (0.1.1)
- resource (2.0.21)
- role (2.0.17)
- servicefabric (0.0.9)
- sql (2.0.19)
- storage (2.0.23)
- vm (2.0.24)

```
14:36:35  ~/.kube 
$ file config
config: ASCII text, with very long lines

14:36:36  ~/.kube 
$ az acs kubernetes get-credentials -n bhargav-k8s -g bhargav-k8s
Merged "bhargav-k8-bhargav-k8s-3fee81mgmt" as current context in /Users/bhargav/.kube/config

14:36:45  ~/.kube 
$ file config
config: ASCII text, with very long lines

14:36:49  ~/.kube 
$ kubectx
bhargav-k8-bhargav-k8s-3fee81mgmt

14:36:51 ~/.kube 
$ az acs kubernetes install-cli
Downloading client to /Users/bhargav/.kube/config from https://storage.googleapis.com/kubernetes-release/release/v1.9.2/bin/darwin/amd64/kubectl

14:37:26  ~/.kube 
$ file config
config: Mach-O 64-bit executable x86_64

14:50:19  ~/.kube
$ ./config version
Client Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.2", GitCommit:"5fa2db2bd46ac79e5e00a4e6ed24191080aa463b", GitTreeState:"clean", BuildDate:"2018-01-18T10:09:24Z", GoVersion:"go1.9.2", Compiler:"gc", Platform:"darwin/amd64"}
error: Error loading config file "/Users/bhargav/.kube/config": yaml: invalid trailing UTF-8 octet
```

**Comment by mboersma** [comment]
This was a mistaken default Path argument that was fixed by #5319, so @bnookala please try the current `az` CLI 2.0.26 or later. And please accept my apologies for this terrible bug. 🤕

**Comment by mboersma** [comment]
@derekbekoe this issue can be closed.
